cmake/Codegen.cmake

```diff
@@ -92,6 +92,9 @@ if(INTERN_BUILD_ATEN_OPS)
       if(EXISTING_ARCH_FLAGS MATCHES ".*compute_90.*")
         list(APPEND ROWWISE_SCALED_MM_FILE_COMPILE_FLAGS "-gencode;arch=compute_90a,code=sm_90a")
       endif()
+      if(EXISTING_ARCH_FLAGS MATCHES ".*compute_100a.*")
```

Comment:
This should be `compute_100` only, not `compute_100a` - namely, the idea here is that PyTorch nightly etc. are not built for all architectures, to save the space, and in particular is not built for `a` variations of architectures.  Instead, when PyTorch built for corresponding non-`a` architecture (that is `100` for `100a`), the NVCC flags for `a` architecture should be applied for `RowwiseScaledMM.cu` file only.